src/main/scala/edu/knowitall/tool/srl/ClearSrl.scala

```diff
@@ -42,7 +42,8 @@ class ClearSrl extends Srl {
   def apply(graph: DependencyGraph): Seq[Frame] = {
     val tree = new DEPTree()
 
-    graph.nodes.zipWithIndex.foreach {
```

Comment:
That works too, and it doesn't require an explicit sort so that's great. Thanks.

I haven't yet reproduced in isolation. This bug is fixable (without code change) by doing an sbt clean and compile, introducing a whitespace change in ClearSrl.scala and issuing compile which recompiles only that file. I guessed that load ordering of classes was changed and affected the way that zipWithIndex (or foreach) works - maybe by way of defining an implicit somewhere. But I haven't discovered the difference in target/ yet, so I don't yet have a clean repro case.